lib/parser.js

```diff
@@ -277,7 +353,8 @@ function parseAuthnrAssertionResponse(msg) {
 module.exports = {
 	parseExpectations,
 	parseClientResponse,
-	parseAttestationObject,
+	// parseAttestationObject,
```

Comment:
Same here - lets not disable old methods, but make them work with the new code until we want to do a major bump anyways

Reply:
My mistake, I have removed it during test on code coverage and never added again.